libs/util/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "@cordis/util",
+  "version": "0.1.0",
+  "description": "Helper REST functions and other utilities for Cordis",
+  "main": "./dist/index.js",
+  "types": "./types/index.d.ts",
+  "scripts": {
+    "lint": "eslint src --ext .ts",
+    "build": "tsc"
+  },
+  "files": [
+    "dist",
+    "types",
+    "!dist/*.tsbuildinfo",
+    "!**/**.map",
+    "!**/*.test.*"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cordis-lib/cordis.git"
+  },
+  "contributors": [
+    "didinele <didinele.dev@gmail.com> (https://github.com/didinele)",
+    "Holo-Buckshot (https://github.com/Holo-Buckshot)",
+    "Nico (https://github.com/zaida04)"
```

Comment:
Feel free to add this to every other package as well, I was gonna do it but forgot.

Reply:
I don't know whether I should do it in this pr or not, but to be safe i'll just do it in a separate PR

Reply:
resolved in https://github.com/cordis-lib/cordis/pull/33/commits/0ed8dcee7774b640a1c7743e2ed91de04ab095a7